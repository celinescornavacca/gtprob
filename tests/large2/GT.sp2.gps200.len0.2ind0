(((((A73,A74),((((A190,A191),A175),A133),(A143,A144))),A4),(((((A178,A179),A15),(((A125,(A194,A195)),((A152,A153),A147)),(A79,A80))),((((A18,((A65,((A138,(A148,A149)),(A127,A128))),(A26,(A30,A31)))),((A52,A53),(A32,A33))),((((((A94,A95),A59),((A165,A166),A28)),((A16,(A112,A113)),A10)),(A0,((A40,(A61,((A171,A172),(A99,A100)))),((A134,A135),((A63,(A156,A157)),(A188,A189)))))),((A180,A181),((A82,A83),A12)))),((((((A192,A193),A39),A7),((A17,(A24,((A105,A106),(A38,(A103,A104))))),(A13,A14))),(A9,(A48,A49))),((A8,(((((((A110,A111),A96),A86),A81),A23),(A117,A118)),(((A77,A78),(A150,A151)),((A87,A88),(A154,A155))))),((((A60,(A91,((A139,A140),A93))),A25),((A97,A98),(A19,A20))),(A121,A122)))))),(((((A6,((((A45,(A66,A67)),A29),((A89,A90),(A176,A177))),(A131,A132))),A1),((A109,(A119,A120)),((A198,A199),((((A182,A183),A107),(A92,(A145,A146))),(A71,((A173,A174),A116)))))),(((A101,A102),A5),(((A43,A44),((A55,A56),((A46,A47),(A184,A185)))),A11))),((((A41,A42),((A162,(A169,A170)),(((A136,A137),(A160,A161)),((A163,A164),A126)))),(A21,(A186,A187))),((((A123,A124),A27),((A37,(A129,A130)),((A64,(A141,A142)),A58))),(((A2,A3),(((A54,(A167,A168)),((((A62,(A75,A76)),A22),(A36,(((A72,(A196,A197)),A57),(A68,A69)))),((A108,(A158,A159)),A70))),(A84,A85))),((A34,A35),((A114,A115),(A50,A51))))))))),((((((B86,B87),(B28,(B142,B143))),(((B46,(B72,B73)),(B33,B34)),((((B30,(B157,B158)),((B63,B64),B59)),B18),((((B93,B94),B20),((B31,(B138,(B145,B146))),((B120,(B165,B166)),B68))),B6)))),(((((B35,B36),((B192,B193),(B115,(B130,B131)))),((B90,B91),B10)),(((((B26,B27),(B17,(B52,B53))),(B163,B164)),(B11,(((B123,B124),B104),B84))),((B167,(B188,B189)),B127))),((((B149,B150),B29),(B121,B122)),(((B178,(B181,B182)),(B176,B177)),B12)))),((B14,(B41,((B185,B186),B69))),B8)),((((((((((B154,B155),B110),(((B95,B96),(B50,B51)),B19)),(((B136,B137),B114),((B159,B160),B76))),(B7,(((B80,B81),(B172,B173)),((B161,B162),B139)))),(((((B132,B133),(B61,B62)),((B125,B126),B37)),(B3,((B66,B67),(B57,B58)))),B1)),B0),(((((B111,B112),(((((B170,B171),B113),B83),(B39,(B118,B119))),B16)),(B23,((B102,B103),B97))),((((B116,B117),B79),(B198,B199)),B9)),(((B152,B153),(B4,(B5,(B65,(((B82,(B140,B141)),(B85,(B134,B135))),(B74,B75)))))),(((B98,B99),((B42,B43),((B44,B45),B22))),((B196,B197),((B32,((B174,B175),B38)),(B24,(B70,B71)))))))),(((B13,(((B60,(B77,B78)),(B108,B109)),((B47,B48),((B92,(B128,B129)),(B105,B106))))),((B25,((B54,B55),((B179,B180),B156))),(B49,(((B194,B195),B187),B144)))),((B15,(B107,(B190,B191))),((B21,((B88,B89),B56)),(B100,B101))))),(((((B183,B184),B151),B40),((B168,B169),(B147,B148))),B2))));
