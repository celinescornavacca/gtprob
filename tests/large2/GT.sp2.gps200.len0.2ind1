(((((((((A100,(A185,A186)),(A51,A52)),(A115,(A141,(A172,A173)))),((((((A19,A20),(A61,A62)),(((A85,A86),A74),(A27,A28))),((A57,(A126,A127)),A0)),((A1,(((((A88,A89),A56),A31),((A40,((A90,A91),(A82,A83))),A17)),(A160,A161))),((A6,(A11,(((A63,A64),(A98,A99)),((A174,A175),A110)))),(((A97,(A145,A146)),(A189,A190)),A5)))),(A2,A3))),((A7,(A15,(A73,(A120,A121)))),(A65,A66))),(((A9,(A13,((((A132,A133),A122),(A116,A117)),(A75,A76)))),((A104,(A113,A114)),A96)),((((A42,A43),((A196,A197),A131)),((((((A181,A182),A180),A108),A77),A30),(A170,A171))),((A49,A50),(A45,((A193,A194),A159)))))),((((A128,A129),((A10,((A29,(A148,A149)),A12)),((A8,((((((A166,A167),A147),A130),A70),(A106,A107)),(((A67,A68),(A24,(A26,(A44,((A183,A184),A125))))),A23))),((A187,A188),(((A123,A124),((A152,((A198,A199),A195)),A105)),(A87,(A92,A93))))))),(((((A134,A135),A25),A16),(((A162,A163),((A157,A158),A138)),((A37,A38),((A191,A192),A41)))),((A143,A144),((A21,(A111,A112)),(A18,(A34,A35)))))),((A4,(A59,A60)),((((((A109,(A168,A169)),A39),A22),(((A150,A151),A142),(((A164,A165),A94),(A55,(A78,A79))))),(((A47,A48),(A53,(A103,(A118,A119)))),((A54,(A101,A102)),(((A69,(A95,(A139,A140))),((A84,(A136,A137)),(A178,A179))),(A80,A81))))),((((A36,(A153,A154)),A33),A32),((A71,A72),(A46,(A176,A177)))))))),(A14,(A58,(A155,A156)))),((((B1,((B165,(B190,B191)),B99)),(B11,B12)),((((((B64,B65),(B86,B87)),((B129,B130),(B123,B124))),((((B109,B110),B100),B48),(B93,B94))),(((B73,(B112,(B170,B171))),(B132,B133)),B25)),(B5,(((B194,B195),B54),(((B115,B116),B104),(B56,B57)))))),(((B0,(B7,(((B28,((B84,B85),((B106,(B122,(B148,(B151,B152)))),((B168,B169),(B184,B185))))),B17),B14))),(((B32,((B46,B47),(B127,B128))),(((((((B36,B37),((((B136,B137),(B76,B77)),(B22,((B146,B147),B45))),(B157,B158))),((B186,B187),(B140,B141))),(((B66,(B78,(B98,(B120,B121)))),(B166,B167)),((((B180,B181),B68),(((B178,B179),(B163,B164)),B125)),B26))),B3),((B34,((B149,B150),B35)),(B142,B143))),(((B58,B59),(B107,B108)),B31))),((((B13,(B20,B21)),((B196,B197),(B89,B90))),((B70,B71),(B63,(B172,B173)))),(B55,(((B182,B183),(B176,B177)),(B159,B160)))))),((((((B79,B80),B69),(B49,B50)),B8),(B2,(((((B38,(B42,B43)),(B27,((B39,B40),(((B138,B139),B119),B95)))),(B29,B30)),((((B53,(B72,(B188,B189))),(B18,B19)),((B61,B62),(B102,B103))),((B51,(B174,B175)),(B23,B24)))),B4))),(((((B96,B97),((B105,(B155,B156)),((B126,(B192,B193)),(B60,(B67,(B153,B154)))))),((((B52,(B198,B199)),(((B134,B135),B41),B33)),(B15,B16)),(((B74,B75),((((B161,B162),B131),B83),B81)),(((B113,B114),(B82,(B91,B92))),B44)))),((B111,(B117,B118)),(B9,B10))),(B6,(((B144,B145),B101),B88)))))));
