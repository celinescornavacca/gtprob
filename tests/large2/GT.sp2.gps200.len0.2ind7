(((((A18,(((A74,A75),A58),(((A180,A181),A179),(A122,(A188,A189))))),((A29,(A41,(A171,A172))),A15)),(((((A7,A8),(A59,(A157,A158))),A2),((A12,(((A56,A57),A53),(A77,A78))),((A85,(A92,((A106,A107),A95))),(A45,(A86,A87))))),(((((A110,A111),A1),(A4,((A155,A156),(((A42,A43),(((((A72,A73),(A66,((((A192,A193),((A153,A154),A132)),(A184,A185)),((A146,A147),A90)))),A37),(A68,(A151,A152))),(A34,(A49,A50)))),(((A163,A164),A46),(A94,(((A140,(A173,A174)),A134),A133))))))),(((A16,(A130,A131)),(A104,A105)),(((((A35,A36),(A161,A162)),(A70,A71)),(((((A83,A84),(A120,A121)),A40),((A93,((A175,A176),A137)),A17)),(A28,(A65,(A167,A168))))),(A91,((A182,A183),(A118,A119)))))),((((A102,A103),((A19,A20),(A22,(A25,A26)))),(((((A32,(A79,A80)),(A159,A160)),(A14,(((A98,A99),((A51,A52),A38)),A27))),((A196,A197),(A125,A126))),((A21,(A96,A97)),A5))),((A6,(A9,((A100,A101),(((A194,A195),A82),A76)))),((A113,A114),(A47,A48))))))),((((((A138,A139),A115),A39),A0),(A13,((((A116,A117),(A129,(A169,A170))),A67),(A186,A187)))),(((((A30,A31),(A54,A55)),((A198,A199),(((A81,((A144,A145),A141)),(A63,A64)),((A190,A191),A60)))),(A3,(((A112,(A123,A124)),(A148,(A177,A178))),(A127,A128)))),(((A23,A24),(((A33,(A135,A136)),((A88,A89),(((A142,A143),A69),A44))),(A10,((A149,A150),(A108,A109))))),(A11,((A61,A62),(A165,A166))))))),((((((B6,(B9,(B43,(B178,B179)))),B0),((((((B29,(B100,B101)),((B136,B137),(B12,((B194,B195),(B198,B199))))),((((((B49,B50),B46),(B54,(B142,B143))),B36),((B128,B129),B31)),(((B124,B125),(((B96,(B156,B157)),B59),(B90,(B172,B173)))),((B62,B63),(((B69,((B94,B95),B74)),B52),(B45,(B154,B155))))))),((B39,(B91,B92)),((B47,B48),B32))),(B2,((((B131,B132),((B60,B61),(B138,B139))),((B111,B112),((B140,B141),B130))),B7))),(((B170,B171),(B41,(B182,B183))),(B162,B163)))),(((B3,((B37,B38),((B174,B175),(B196,B197)))),((((B70,B71),(B104,B105)),B14),(B44,((B72,B73),B53)))),(((B190,B191),B23),B5))),(((((((((B121,(B168,B169)),(B113,B114)),((B134,B135),B30)),(B56,(B57,(B87,B88)))),(B21,B22)),((B176,B177),B10)),((B27,B28),(B16,((B78,B79),(((B158,B159),((B82,B83),((B102,B103),B93))),((B144,B145),(B75,(B106,B107)))))))),(B8,((((B76,B77),(((B98,B99),(B188,B189)),((B164,B165),B68))),(B126,B127)),B11))),((B97,(B148,B149)),((((B66,B67),B42),(((B146,B147),(B108,B109)),((B186,B187),B133))),B4)))),((((B33,B34),(B15,(B80,B81))),(B19,(B89,(B184,B185)))),(((((((B152,B153),(((((B117,B118),B13),(B26,(B35,((B180,B181),B55)))),(B17,B18)),(B58,(((B115,B116),(B122,B123)),B84)))),((B85,B86),B25)),(B150,B151)),((((B192,B193),B51),B20),B1)),(((B160,B161),((B166,B167),B110)),(B119,B120))),(((B64,B65),B40),B24)))));
