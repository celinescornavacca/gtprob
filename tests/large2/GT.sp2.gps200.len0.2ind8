((((((A20,((A198,A199),A193)),((A94,(A108,A109)),(A161,A162))),(A18,(A130,A131))),A1),(((((((A105,(A138,A139)),(A169,A170)),((A110,A111),A19)),((((A34,A35),(((A84,A85),(A183,A184)),A32)),(A25,((A103,A104),((A155,A156),(A118,(A175,A176)))))),A7)),A3),((((((((A69,A70),(A12,((A43,A44),((A58,((A144,A145),(A101,A102))),((((A181,A182),(((A159,A160),(A100,(A189,A190))),A74)),A57),(A136,A137)))))),(A13,(((((A106,A107),A97),(A121,A122)),(A53,(A112,A113))),((A126,A127),(A187,A188))))),(((A153,A154),(A173,A174)),(A27,(A71,(A185,A186))))),((((A150,A151),A52),(A28,(A194,A195))),(A15,(A46,((A76,(A146,A147)),A75))))),((A17,((A191,A192),(A81,(A88,A89)))),(((A196,A197),A36),((A148,A149),A93)))),((A165,A166),((((A179,A180),A125),A72),A10))),((((((((A171,A172),A152),(A177,A178)),A90),A5),(A9,((A21,A22),(A49,(A140,A141))))),(((A31,((A67,A68),A33)),((((A91,A92),A54),(A82,A83)),((A41,(A61,A62)),A37))),(A42,(A73,(A77,A78))))),((A40,(A114,A115)),(A117,(A163,A164)))))),(((((A6,((A29,A30),A23)),(A123,A124)),(((A11,(A63,A64)),((A95,A96),(A38,A39))),A2)),((A142,A143),(A4,(((A45,(A59,A60)),((A65,A66),A14)),(((A157,A158),(A48,(A55,A56))),A16))))),((((((A47,(A134,A135)),((A79,A80),(A116,(A167,A168)))),(A26,((A98,A99),A51))),(A128,A129)),((A86,A87),((((A119,A120),(A50,(A132,A133))),A24),A8))),A0)))),(((((((B158,B159),(B148,B149)),(((((B93,(B135,B136)),B39),(B46,(B47,(B60,B61)))),(B181,B182)),(((B48,(B75,(B175,B176))),(B130,B131)),(((B187,B188),(B168,B169)),(B109,((B142,B143),B132)))))),((((B146,B147),B49),(((B194,B195),B32),(B121,B122))),(B22,((B74,(B94,B95)),(((B190,B191),(B196,B197)),(((B137,B138),(B110,B111)),B25)))))),(((B43,B44),(((B198,B199),(B112,B113)),(B141,(B150,B151)))),(B54,B55))),(((((((B33,((B170,B171),B63)),(B68,B69)),(B62,(B154,B155))),(B99,B100)),((B1,((B12,(B126,B127)),(B76,B77))),(B98,(B128,B129)))),((B16,((B50,(B101,B102)),(B56,B57))),(B4,B5))),(((B2,(((B105,B106),(B164,B165)),B37)),((B10,(((((B144,B145),B125),B67),(B84,B85)),(B42,(B177,B178)))),((B51,(B103,B104)),(B38,(B72,B73))))),((((B24,(B133,B134)),((B89,B90),(B119,B120))),(B82,B83)),((((B52,B53),((B64,B65),(B58,(B86,B87)))),((((B183,B184),B174),(B14,B15)),B3)),(((B20,(B172,B173)),((((B96,(B116,B117)),(B70,B71)),B17),(B123,B124))),((B162,B163),B13))))))),((B0,((((B28,B29),(((((B179,B180),B66),(B185,B186)),((B160,B161),B88)),B23)),B7),((((B26,(B30,B31)),(((B35,B36),((((B192,B193),B189),(B114,B115)),(B80,B81))),((B91,B92),(B118,(B152,B153))))),((B97,(B139,B140)),(B27,(B166,B167)))),B8))),((((B18,B19),B9),(((B11,(B34,((B156,B157),B59))),((B21,(B40,B41)),(B107,B108))),B6)),(B45,(B78,B79))))));
