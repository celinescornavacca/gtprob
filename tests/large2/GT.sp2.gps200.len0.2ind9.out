-1948.26439492
